service/ioc-start.sh

```diff
@@ -14,6 +14,12 @@ done
 
 start_afc_ioc=false
 
+synthesis="$(decode-reg build_info -q --slot ${devslot})"
+if [ -z "$synthesis" ]; then
+    decode-reg reset --slot ${devslot}
+    synthesis="$(decode-reg build_info -q --slot ${devslot})"
+fi
+
 case "$(decode-reg build_info -q --slot ${devslot})" in
```

Comment:
Did you intend to use `$synthesis` in the `case` argument?

Reply:
I did, yes. Thanks!